lms/envs/production.py

```diff
@@ -56,6 +56,18 @@ def get_env_setting(setting):
     ENV_TOKENS = __config__
     AUTH_TOKENS = __config__
 
+# A file path to a YAML file from which to load all the code revisions currently deployed
+REVISION_CONFIG_FILE = get_env_setting('REVISION_CFG')
```

Comment:
@cpennington 
this line raising ImproperlyConfigured() Error while provisioning as REVISION_CFG is not available env variables.

Reply:
@HammadAhmadWaqas Can you post specifically the context where that's failing? There's a change in `edx/configuration` that should be adding that variable to `edxapp_env`, but I've been getting other reports that it's not working, so I want to try and narrow down what's going on.